### PR TITLE
MFH

### PR DESCRIPTION
**Pull request by bjori**
We really should do this more frequently.

### CODE REVIEW

File: src/mongoc/mongoc-topology-scanner-private.h [hunk 1]
```diff
@@ -94,8 +93,7 @@ mongoc_topology_scanner_node_destroy (mongoc_topology_scanner_node_t *node,
 
 void
 mongoc_topology_scanner_start (mongoc_topology_scanner_t *ts,
-                               int32_t timeout_msec,
-                               bool obey_cooldown);
+                               int32_t                    timeout_msec);
```
Comment (ajdavis) [inline src/mongoc/mongoc-topology-scanner-private.h]: obey_cooldown will be *deleted* by this merge?

File: tests/test-mongoc-client.c [hunk 2]
```diff
@@ -686,6 +927,15 @@ test_mongoc_client_ipv6 (void)
 void
 test_client_install (TestSuite *suite)
 {
+   bool local;
+   local = !getenv ("MONGOC_DISABLE_MOCK_SERVER");
```
Comment (ajdavis) [inline tests/test-mongoc-client.c]: This was never right, let's not reintroduce the "local" bool or the MONGOC_DISABLE_MOCK_SERVER env var. We should always run the mock server tests.

File: src/mongoc/mongoc-topology-scanner-private.h (hunk 1 above)
Comment (bjori) [inline src/mongoc/mongoc-topology-scanner-private.h]: Fixed nice catch

File: tests/test-mongoc-client.c (hunk 2 above)
Comment (bjori) [inline tests/test-mongoc-client.c]: fixed

**Comment by ajdavis** [comment]
Thank you!!

**Pull request merged**